internal/impl/deploy.go

```diff
@@ -86,6 +88,18 @@ func Deploy(ctx context.Context, configFilename string) error {
 	if config.Repo == "" {
 		fmt.Fprintln(os.Stderr, "No container repo specified in the config file. The container image will only be accessible locally. See `weaver kube deploy --help` for details.")
 	}
+	if config.BuildTool == "" {
```

Comment:
What do you think about rewriting this as:

`
 switch config.BuildTool {
       case "docker", "podman":
       case "": // not specified
               config.BuildTool = defaultBuildTool
       default:
               return fmt.Errorf("unsupported build tool: %s", config.BuildTool)
       }
`

Reply:
maybe the error message should be "return fmt.Errorf("unsupported build tool: %s; supported tools are `docker` and `podman`", config.BuildTool)"